recipes/emacs-25-pre-travis.rb

```diff
@@ -0,0 +1,7 @@
+recipe 'emacs-25-pre-travis' do
+  tar_gz 'https://github.com/rejeep/evm/releases/download/v0.6.1/emacs-25-pre-travis.tar.gz'
```

Comment:
Change this to `https://github.com/rejeep/evm/releases/download/v0.8.0/emacs-25-pre-travis.tar.gz `. I will release `v0.8.0` with this change and attach the binary to that release.